app/models/category_bill.rb

```diff
@@ -0,0 +1,4 @@
+class CategoryBill < ApplicationRecord
+  belongs_to :category
+  belongs_to :bill
+end
```

Comment:
- [ ]  I noticed that in you seem to have missed validating the category_id and bill_id_columns. Kindly include validations for these column as well.